shopping-cart/src/components/ProductInCart.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/prop-types */
+export default function ProductInCart({
+  product,
+  onChangeProductQuantity,
+  onProductDelete,
+}) {
+  console.log(product.name);
+  return (
+    <div className="p-4 border-2 rounded-md">
+      <img src={product.imageUrl} alt={product.imageAlt} />
+      <h3>{product.name}</h3>
+      <h5>
+        BDT: <b>{product.unitPrice}</b>
+      </h5>
+      <div className="space-x-2">
+        <label>Qty: </label>
+        <select
```

Comment:
better if + - with icons instead of dropdown. If dropdown make it dynamic.

Reply:
Implemented + -  with material icons